mrp_bom_widget_section_and_note_one2many/views/view_mrp_bom.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+Copyright (C) 2022 - Today: GRAP (http://www.grap.coop)
+@author: BADEP
+@author: Quentin DUPONT (quentin.dupont@grap.coop)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo>
+
+  <record id="mrp_bom_form_view_section" model="ir.ui.view">
+      <field name="model">mrp.bom</field>
+      <field name="inherit_id" ref="mrp.mrp_bom_form_view" />
+      <field name="arch" type="xml">
+        <!-- Add new page -->
+          <xpath expr="//page[1]" position="before">
+            <page
+                    string="Components with sections and notes"
```

Comment:
Not a blocking point

Maybe a taste question but when you have a huge installation with many modules and then many tabs, having long strings in tab names are not an advantage (think usability on mobile)

```suggestion
                    string="Components and sections"
```

Reply:
fine for me !

Reply:
change done in new commit :)